packages/ic-error-types/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "ic-error-types"
+version = "0.0.0"
```

```suggestion
version = "0.1.0"
```
Just a quick note: since this crate is now being published, it’d be good to update the version to 0.1.0 before merging. The changelog should also be updated.

---

Yes, thanks for the reminder!